src/deluge/io/midi/midi_follow.cpp

```diff
@@ -146,8 +147,9 @@ MidiFollow::getModelStackWithParam(ModelStackWithThreeMainThings* modelStackWith
                                    int32_t xDisplay, int32_t yDisplay, int32_t ccNumber, bool displayError) {
 	ModelStackWithAutoParam* modelStackWithParam = nullptr;
 
-	bool isUISessionView =
-	    (getRootUI() == &performanceSessionView) || (getRootUI() == &sessionView) || (getRootUI() == &arrangerView);
+	bool isUISessionView = (getRootUI() == &performanceSessionView) || (getRootUI() == &sessionView)
```

Comment:
this should be renamed, seems like it's just if you're not in a clipview? Which would be easy if the UI stack was being used